docs/en/ep1/index.html

```diff
@@ -280,12 +280,7 @@ <h3 class="no_toc d-none d-md-block nomargin">for the Public Administration</h3>
 <div class="container">
   <div class="row">
     <div class="col col-lg-8 my-5">
-      <h1 class="otherpages">Episode 1</h1>
-<h2 class="home">Human Error</h2>
-
-In this story we introduce the concept of <b>digital security</b> linked to the correct behavior of people in a working and private environment. <b>About <a href="https://www.social-engineer.org/social-engineering/social-engineering-infographic/">66 percent of cyber attacks</a> occur "between the computer and the chair"</b>, that is, they do not depend on technical weaknesses in computer systems but on human error. This means that employees are an important link for protecting corporate data, digital security is therefore the responsibility of each of us. The aim of the course is to provide <b>basic tools to increase safety</b>, also through prudent individual behavior.
-
-In this story, for example, we learn what can happen (and it really happens in the real world!) When we don't take the necessary precautions <b>in the management of work documents</b>, which may contain valuable information...
+      
```

Comment:
In realtà con l'aggiornamento di Hugo si è persa roba, devo vedere perché.

Reply:
Ok, sistemato. C'erano dei file `intro.md` che in realtà erano puro HTML, bastava aggiungere:
```
---
---
```
in testa al file.